spec_linux.go

```diff
@@ -150,3 +153,16 @@ type Resources struct {
 	// Network restriction configuration.
 	Network Network `json:"network"`
 }
+
+type LinuxState struct {
```

Comment:
I'd rather add this to a new `state_linux.go`, since it's not part of the spec.  It's still in the `specs` package, so you could still use `spec_linux.go`-defined types in the `LinuxState` structure.